components/ui/skeleton.tsx

```diff
@@ -0,0 +1,18 @@
+import { cn } from '@/lib/utils/tailwind/cn';
+
+function Skeleton({
+  className,
+  ...props
+}: React.HTMLAttributes<HTMLDivElement>) {
+  return (
+    <div
+      className={cn(
+        'animate-pulse rounded-md bg-purple-100 dark:bg-gray-600',
+        className
```

Comment:
Nice approach, but do all the skeleton components in our project will use the 'animate-pulse` animation? 

If so this is fine, but If not maybe give the components  less generic name or accept animation as props?

What do you think?

Reply:
If you'll use **THIS**  specific skeleton then YES! This is the official component from [shadcn/ui](https://ui.shadcn.com/docs/components/skeleton)
You can create a custom one if you don't like the animation